lib/ansible/module_utils/tacp.py

```diff
@@ -196,9 +199,21 @@ def is_valid_uuid(uuid_to_test, version=4):
 
 def api_response_to_dict(api_response):
     if type(api_response) != ApiException:
-        message_str = api_response.message
-    else:
-        message_str = str(api_response)
+        if type(api_response) == str:
+            message_str = api_response
+            message_str = message_str.replace('\n', '').replace('null', 'None')
+        elif "message" in dir(api_response):
+            message_str = api_response.message
+        else:
+            attrs = [key for key in api_response.__dict__.keys() if not key.startswith("__") and key != "discriminator"]
+            api_dict = {}
+            for attr in attrs:
+                val = str(api_response.__dict__[
+                    attr]).replace('\n', '').replace('null', 'None')
+                extra_space_pattern = ",\\s{2,}"
```

Comment:
Can we get some example for an api response in the docstring or in a comment? This way we could easily understand the logic here